addons/pandora/api.gd

```diff
@@ -15,7 +15,7 @@ signal import_calculation_failed(reason: String)
 signal import_progress
 
 var _context_manager: PandoraContextManager
-var _storage: PandoraJsonDataStorage
```

Comment:
Unecessarily leaked an implementation. We should not care about json at this point.